src/serializer/serializer.js

```diff
@@ -603,7 +678,7 @@ export class Serializer {
         if (delayReason) return delayReason;
       }
     } else if (val instanceof FunctionValue) {
-      if (!this.firstFunctionUsages.has(val)) this.firstFunctionUsages.set(val, this._getBodyReference());
+      if (!this.firstFunctionUsages.has(val)) this.firstFunctionUsages.set(val, new BodyReference(this.body, this.body.length));
```

Comment:
Figuring out that this a non behavior changing cosmetic change caused me to slow down.